src/main/java/com/bina/varsim/fastqLiftover/readers/ARTFastqAlnReader.java

```diff
@@ -23,11 +23,14 @@ public ARTFastqAlnReader(final BufferedReader alnR, final BufferedReader fastqBr
     public SimulatedRead getNextRead() throws IOException {
         SimulatedRead read;
         final String nameLine = fastqBr.readLine();
-        if (nameLine == null) return null;
+        //TODO: add logging
```

Comment:
Additionally, it would be good to count lines in the reader so that when bad things happen, can print out the line number where this happened.